certified-connectors/SigmaConsoCR/apiDefinition.swagger.json

```diff
@@ -0,0 +1,292 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "title": "Sigma Conso CR",
+      "description": "Sigma Conso Consolidation & Reporting Connector provides a set of APIs to interact with your Consolidation & Reporting instances. Easily transfer data, trigger actions and automate workflows in Sigma Conso. Go beyond the limits of traditional CPM solutions.",
+      "version": "1.0",
+      "contact": {
+        "name": "Sigma Conso",
+        "url": "https://www.sigmaconso.com/",
+        "email": "support@sigmaconso.com"
+      }
+    },
+    "host": "x",
```

Comment:
@tve-sigmaconso Providing `x` would be a syntax error in the build system, Can you please update it with `"yourhost.yourdomain.com"`

Reply:
This is fixed in commit 47bbf27e.